src/main/webapp/repo-bootstrap/global/configuration/samples/sample-site-policy-config.xml

```diff
@@ -44,12 +44,16 @@
     -->
 
     <!-- Example: only allow images of less than 1 MB -->
+    <!-- disable svg files -->
     <statement>
         <target-path-pattern>/static-assets/images/.*</target-path-pattern>
         <permitted>
             <maximum-file-size>1000000</maximum-file-size>
             <mime-types>image/*</mime-types>
         </permitted>
+        <denied>
```

Comment:
@phuongnq changes in samples need to be added to the UM, see [here](https://github.com/craftercms/studio/blob/develop/src/main/resources/crafter/studio/upgrade/pipelines.yaml#L365) for an example and let me know if you have questions about this

Reply:
Updated UM. Thanks! (Let me know if I missed any configuration)